ui/apps/platform/src/Containers/Dashboard/PatternFly/DashboardPage.tsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import { Divider, PageSection, Text, Title } from '@patternfly/react-core';
+
+function DashboardPage() {
+    return (
+        <>
+            <PageSection variant="light">
+                <Title headingLevel="h1">Dashboard</Title>
+                <Text>Review security metrics across all or select resources</Text>
+            </PageSection>
+            <Divider component="div" />
+            <PageSection />
```

Comment:
Although this follows recent pattern, the lead time give a good opportunity to double-check with UX team and also PatternFly team the pros and cons for accessibility and semantics of multiple `PageSection` elements which renders `section` elements to obtain the desired visual appearance.

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/section

> Sections should always have a heading, with very few exceptions.

Reply:
Good call, it has felt a little odd using that component as a way to split up the page. Also in the link you provided:
| If you are only using the element as a styling wrapper, use a [<div>](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/div). A rule of thumb is that a <section> should logically appear in the outline of a document.

I checked and unfortunately the `PageSection` component doesn't have a prop that lets us specify the underlying element like some other components do. It might be worth asking on the PF Slack if there is something that fits this role, if not I suppose it shouldn't be too tough to build our own component that applies the styles without the semantics.